modules/migration-mtc-release-notes-1-8-3.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-3_{context}"]
+= {mtc-full} 1.8.3 release notes
+
+[id=technical-changes-1-8-3_{context}]
+== Technical changes
+
+{mtc-full} ({mtc-short}) 1.8.3 has the following technical changes:
+
+.Support added for {oadp-short} 1.3
+
+{mtc-short} 1.8.3 adds support to {oadp-first} as a dependency of {mtc-short} 1.8.z. link:https://issues.redhat.com/browse/MIG-1520[(MIG-1520)]
```

Comment:
```suggestion
{mtc-short} 1.8.3 adds support to use {oadp-first} as a dependency of {mtc-short} 1.8.z. link:https://issues.redhat.com/browse/MIG-1520[(MIG-1520)]
```

I don't know the "Docs rules", so if `use` isn't preferred, perhaps, `install` or whatever is more appropriate.